lib/cylc/review.py

```diff
@@ -661,6 +661,8 @@ def viewsearch(self, user, suite, path=None, path_in_tar=None, mode=None,
     def view(self, user, suite, path, path_in_tar=None, mode=None,
              no_fuzzy_time="0"):
         """View a text log file."""
+        if re.findall(r"log\..*\d\d\s\d\d$", path):
+            path = path.replace(' ', '+')
```

Comment:
Replacing space with plus in path?  (This doesn't seem to gel with the description of the problem: failing to display logs with `+` in the name ... however, I haven't gone any deeper than that!)

Reply:
I may be misunderstanding how the CherryPi app is working, but the `CylcReviewService.view` method is being called (by the GET?) with `+` removed from filenames in the form `log.YYYYMMDDTHHMM+ZZ`. This is a crude hack because I don't want to spend too long understanding what CherryPi is up to.

Reply:
OK fair enough (I certainly haven't looked at what CherryPy is doing!)